locales/en/plugin__dfr-console.json

```diff
@@ -32,10 +32,12 @@
   "Deployment Name": "Deployment Name",
   "An error occurred": "An error occurred",
   "Attach": "Attach",
-  "Create NamespaceStore ": "Create NamespaceStore ",
-  "Represents an underlying storage to be used as read or write target for the data in the namespace buckets.": "Represents an underlying storage to be used as read or write target for the data in the namespace buckets.",
-  "Namespace store name": "Namespace store name",
-  "A unique name for the namespace store within the project": "A unique name for the namespace store within the project",
+  "Create Data Resource": "Create Data Resource",
+  "Represents an underlying storage to be used as read or write target for the data in buckets.": "Represents an underlying storage to be used as read or write target for the data in buckets.",
+  "Missing a required property - Target bucket": "Missing a required property - Target bucket",
+  "Missing required property - Access key or Secret key": "Missing required property - Access key or Secret key",
```

Comment:
no need to have to separate messages... check once what message we show at other places in odf-console we can't use that message only...